src/components/ImageGeneration/QueueSnackbar.tsx

```diff
@@ -0,0 +1,160 @@
+      >
+        <div className="flex items-center basis-20 py-2 pl-1">
+          {queueStatus && (
+            <GenerationStatusBadge status={queueStatus} count={count} quantity={quantity} />
+          )}
+        </div>
+        <div className="flex flex-col gap-1 items-center justify-center flex-1 py-2">
+          <Text weight={500} className="flex gap-1 items-center">
+            {!!queued.length ? (
+              <>
+                {(queueStatus === GenerationRequestStatus.Pending ||
+                  queueStatus === GenerationRequestStatus.Processing) && (
+                  <>
+                    <span>{`${queued.length} job${queued.length > 0 && 's'} in `}</span>
+                    {includeQueueLink ? (
+                      <Text
+                        inline
+                        variant="link"
+                        className="cursor-pointer"
+                        onClick={() => generationPanel.setView('queue')}
+                      >
+                        queue
+                      </Text>
+                    ) : (
+                      <span>queue</span>
+                    )}
+                  </>
+                )}
+                {queueStatus === GenerationRequestStatus.Succeeded && 'All jobs complete'}
+                {queueStatus === GenerationRequestStatus.Error && 'Error with job'}
+              </>
+            ) : (
+              `${requestsRemaining} jobs available`
+            )}
+          </Text>
+          <div className="flex gap-2 w-full justify-center">
+            {slots.map((slot, i) => {
+              const item = queued[i];
+              const colors = theme.fn.variant({
+                color: item ? generationStatusColors[item.status] : 'gray',
+                variant: 'light',
+              });
+
+              const progress = !item ? 0 : (item.count / item.quantity) * 100;
+              return (
+                <Progress
+                  key={i}
+                  color={item ? generationStatusColors[item.status] : 'gray'}
+                  radius="xl"
+                  value={progress}
+                  h={6}
+                  w="100%"
+                  maw={32}
+                  style={{ backgroundColor: item ? colors.background : undefined }}
+                  className="flex-1"
+                />
+              );
+            })}
+          </div>
+        </div>
+        <div className="flex items-center justify-end basis-20 py-1">
+          {images.map((image, i) => (
+            <Card
+              key={image.id}
+              withBorder
+              radius="md"
+              p={0}
+              style={{
+                zIndex: i,
+                marginRight: i < images.length - 1 ? `-33.33%` : undefined,
+                height: 42,
+              }}
+            >
+              {/* eslint-disable-next-line jsx-a11y/alt-text, @next/next/no-img-element */}
+              <img alt="" src={image.url} className="max-h-full" />
```

Comment:
Should we be using `EdgeMedia` here?

Reply:
`EdgeMedia` doesn't work with orchestrator images